Handle 4th axis (A)

dguerizec opened the pull request:
With some small changes, this post processor can now handle [4 axis on grbl](https://github.com/gnea/grbl-Mega/pull/21)

dguerizec committed:
Handle 4th axis (A)

Strooom commented:
Hi, thanks for your improvements. I took a look in order to merge the code, but there is not enough comments for me to understand what/why/how you are adding.
Please add some comments to the lines you added so I can easily understand them.
Thank you

dguerizec committed:
Fixed 4th axis for milling operations

dguerizec commented:
Hello, sorry for the delay.
I just fixed the 4th axis code for milling operations (it was only working for engraving on a cylinder).
I added comments, but I'm not a Fusion post expert.
